R/shiny.R

```diff
@@ -846,6 +846,17 @@ ShinySession <- R6Class(
         registerDataObj = function(name, data, filterFunc) {
           .subset2(self, "registerDataObj")(ns(name), data, filterFunc)
         },
+        sendInsertTab = function(inputId, liTag, divTag, menuName,
+                                 target, position, select) {
+          .subset2(self, "sendInsertTab")(ns(inputId), liTag, divTag, 
```

Comment:
Instead of reimplementing these, can you just have the "real" implementations wrap the `inputId` with `session$ns()`?